core/src/main/java/org/infinispan/stats/impl/ClusterCacheStatsImpl.java

```diff
@@ -261,6 +272,29 @@ public long getMisses() {
       return getStatAsLong(MISSES);
    }
 
+   @ManagedAttribute(description = "Cluster wide approximate number of entries in the cache, including passivated entries and ignoring copies",
+         displayName = "Cluster wide approximate number of cache entries"
+   )
+   public long getApproximateEntries() {
+      return getStatAsInt(APPROXIMATE_ENTRIES);
```

Comment:
```suggestion
      return getStatAsLong(APPROXIMATE_ENTRIES);
```
same for the methods below.

Reply:
Good catch!